src/main/java/com/conveyal/r5/GraphQLSchema.java

```diff
@@ -697,6 +699,65 @@ public GraphQLSchema(PointToPointQuery profileResponse) {
                 .build())
             .build();
 
+        transitEdgeInfoType = GraphQLObjectType.newObject()
+            .name("TransitEdgeInfo")
+            .description("Represents one transit network link")
+            .field(GraphQLFieldDefinition.newFieldDefinition()
+                .name("id")
+                .type(new GraphQLNonNull(Scalars.GraphQLString))
+                .description("transit edge id composed of stop IDs and route ID")
```

Comment:
If this needs to be unique, note that two stop IDs and a route ID will not uniquely identify a transit link - you'd probably need to use a trip ID.